cdc/sink/mysql.go

```diff
@@ -513,17 +512,15 @@ var GetDBConnImpl = getDBConn
 func getDBConn(ctx context.Context, dsnStr string) (*sql.DB, error) {
 	db, err := sql.Open("mysql", dsnStr)
 	if err != nil {
-		return nil, errors.Annotate(
-			cerror.WrapError(cerror.ErrMySQLConnectionError, err), "Open database connection failed")
+		return nil, cerror.ErrMySQLConnectionError.Wrap(err).GenWithStack("Open database connection failed")
```

Comment:
Why is it different from the error message above. 😆

Reply:
errors.Annotate will return an error that is not a RFCCode(err), it is hard to detect its type.

Reply:
I mean, why not use the same sentence for the same error, when it looks like the stack information is just in a different order of words?

Reply:
"Open database connection failed" and "fail to open MySQL connection"

Reply:
got it